lib/src/chart/line_chart/line_chart.dart

```diff
@@ -213,10 +215,12 @@ class _LineChartState extends AnimatedWidgetBaseState<LineChart> {
         _showingTouchedTooltips.add(MapEntry(0, sortedLineSpots));
       });
     } else {
-      setState(() {
-        _showingTouchedTooltips.clear();
-        _showingTouchedIndicators.clear();
-      });
+      if (!widget.stickyTouchedIndicators) {
```

Comment:
You were supposed to revert all changes about the sticky headers.

Reply:
Ah I forgot about that. I need this feature in my app and I had the changes locally for me.